tasks/pkg-debian/install-apm-inject.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Set APM injection languages and versions for Debian
+  set_fact:
+    agent_dd_apm_install_pkgs: "{{ (agent_dd_apm_install_pkgs |
+      default([], true)) + ['datadog-apm-library-' + (item | regex_replace('[:]', '='))] }}"
+  loop: "{{ pre_processed_datadog_apm_libraries }}"
+
+- name: Install APM Inject Library
+  apt:
+    name: 'datadog-apm-inject'
+    state: latest # noqa package-latest
+    update_cache: true
+    cache_valid_time: "{{ datadog_apt_cache_valid_time }}"
```

Comment:
Let's use `when: not ansible_check_mode` here as well.

Reply:
Done